### DISCUSSION TITLE
Can i spawn helicopters via console commands?

### DISCUSSION THREAD

**Discussion by Dife2K** [TEHE-Studios/ExpandedHelicopterEvents#122] [Q&A]
Is it possible to remove the random encounters and only spawn a helicopter via console like the vanilla one or items? Would like to make it roleplayable! :)

**Comment by Chuckleberry-Finn** [comment]
You can spawn them via number keys and disable random using to the Twitch Integration mod.

**Reply by TURRETCUBE123** [reply]
can't use the num keys, is there a command i gotta put in? where is the console?
